kong/response.go

```diff
@@ -34,6 +34,21 @@ func hasError(res *http.Response) error {
 		return nil
 	}
 
-	body, _ := io.ReadAll(res.Body) // TODO error in error?
+	body, err := io.ReadAll(res.Body) // TODO error in error?
+	if err != nil {
+		return fmt.Errorf("failed to read error body: %w", err)
+	}
 	return NewAPIError(res.StatusCode, messageFromBody(body))
 }
+
+func hasErrorRaw(res *http.Response) error {
+	if res.StatusCode >= 200 && res.StatusCode <= 399 {
+		return nil
+	}
+
+	body, err := io.ReadAll(res.Body) // TODO error in error?
```

Comment:
Do we need this todo? If so let's create an issue for that, tracking it. Otherwise let's remove it

Reply:
nope